CMakeLists.txt

```
@@ -84,7 +84,19 @@ else()
 endif()
 
 # Python.h needed by torch headers.
-find_package(Python3 REQUIRED COMPONENTS Development)
+find_package(Python3 REQUIRED COMPONENTS Development.Module)
+
+set(RHEL_BUILD OFF)
+set(LIB_DIR "lib")
+set(PY_INSTALL_PATH "/usr/local/lib/python3.10/dist-packages")
```

I'm not insist on it but with it we shouldn't carry about versions match as long as we aligned with DLFW.
```suggestion
set(PY_INSTALL_PATH ${Python_SITEARCH})
```

---

Right, and I've checked that `sysconfig.get_path('platlib')` returns a value--and it does. However, CMake still fails to set the return values for some reason.

That said, after more consideration, I don't think we can go with this approach because the `PY_INSTALLPATH` we are setting is relative to the pytorch container we are copying files out of, not the container we are in when we are building. To illustrate why this is an issue, even if CMake was properly setting the value of `Python_SITEARCH`, relative to the build container, it would be:
`/opt/python/cp310-cp310/lib/python3.10/site-packages`
However, relative to the pytorch container it would need it to be:
`/opt/_internal/cpython-3.10.13/lib/python3.10/site-packages`